cmd/tracee/cmd/root.go

```diff
@@ -115,6 +115,19 @@ func initCmd() error {
 		"[file|dir]\t\t\t\tPath to a policy or directory with policies",
 	)
 
+	// Runtime flags
+
+	rootCmd.Flags().StringArrayP(
+		flags.RuntimeFlag,
+		"r",
+		[]string{"workdir=" + flags.WorkdirDefault},
```

Comment:
Is it really a workdir? I mean, Tracee is gonna change the cwd to it?

Reply:
Tracee store pids, and any artifacts it captures there, so workdir makes more sense than install path